AuxTel/Troubleshooting/ATCS/ATDome-Top-Comm-Error.rst

```diff
@@ -0,0 +1,109 @@
+the script fails showing a failure to communicate with the dome top-end where the shutter control resides. 
+
+.. _Top-Comm-Error-Error-Diagnosis:
+
+Error diagnosis
+===============
+
+.. This section should provide simple overview of known or suspected causes for the error.
+.. It is preferred to include them as a bulleted or enumerated list.
+.. Post screenshots of the error state or relevant tracebacks.
+
+When the command to the shutter is run within the script, an error is received in LOVE showing a communication error:
+
+.. code-block:: bash
+
+    ack=<SalRetCode.CMD_FAILED: -302>
+
+    result="Failed: Command SC returned 1 lines instead of 0; read: ' Top Comm Error\\r
+
+Or the full traceback may be displayed as follows:
+
+.. code-block:: python
+    :linenos:
+
+    2023/06/29 23:13:03 TAI
+    Error in run
+    Traceback (most recent call last):
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/base_script.py", line 603, in do_run
+    await self._run_task
+    File "/net/obs-env/auto_base_packages/ts_standardscripts/python/lsst/ts/standardscripts/auxtel/shutdown.py", line 62, in run
+    await self.attcs.shutdown()
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 738, in shutdown
+    raise e
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 732, in shutdown
+    await self.close_dome()
+    File "/net/obs-env/auto_base_packages/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 946, in close_dome
+    await self.rem.atdome.cmd_closeShutter.set_start(
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/topics/remote_command.py", line 416, in set_start
+    return await self.start(timeout=timeout, wait_done=wait_done)
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/topics/remote_command.py", line 487, in start
+    return await cmd_info.next_ackcmd(timeout=timeout)
+    File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-6.0.0/lib/python3.10/site-packages/lsst/ts/salobj/topics/remote_command.py", line 191, in next_ackcmd
+    raise base.AckError(msg="Command failed", ackcmd=ackcmd)
+    lsst.ts.salobj.base.AckError: msg='Command failed', ackcmd=(ackcmd private_seqNum=71336703, ack=<SalRetCode.CMD_FAILED: -302>, error=1, 
+    result="Failed: Command SC returned 1 lines instead of 0; read: ' Top Comm Error\\r n>'")
+
+
+.. _Top-Comm-Error-Procedure-Steps:
+
+Procedure Steps
+===============
+
+This is a hardware issue related to a lost communication with the top-end or rotating part of the dome. 
+This can happen after a dome cabinets reboot, a power-off-on sequence or a power outage, that, not done 
```

Comment:
It can also happen with a local network failure since the boxes rely on wifi. I think this sentence  has a few too many commas....you might be able to clean it up with something like "after an incorrect power sequence or network communication loss".